content/04.baselines.md

```diff
@@ -1 +1,10 @@
 ## Tip 2: Use traditional methods to establish performance baselines
+
+Before diving into a fancy thousand-layer neural network, always implement at least a simple model to establish an adequate performance baseline. 
+For example, researchers can build multinomial logistic regression or random forests models using the same software framework that is being used for DL and evaluate its classification performance. 
```

Comment:
"forests" -> "forest"

Reply:
Thanks for the edit suggestion. I'll make the change accordingly.